receiver/opencensusreceiver/ocmetrics/opencensus.go

```diff
@@ -113,13 +114,17 @@ func (ocr *Receiver) processReceivedMetrics(longLivedRPCCtx context.Context, ni
 func (ocr *Receiver) sendToNextConsumer(longLivedRPCCtx context.Context, md consumerdata.MetricsData) {
 	// Do not use longLivedRPCCtx to start the span so this trace ends right at this
 	// function, and the span is not a child of any span from the stream context.
-	_, span := obsreport.StartMetricsReceiveOp(
+	tmpCtx := obsreport.StartMetricsReceiveOp(
 		context.Background(),
 		ocr.instanceName,
 		receiverTransport)
 
 	// If the starting RPC has a parent span, then add it as a parent link.
-	obsreport.SetParentLink(longLivedRPCCtx, span)
+	obsreport.SetParentLink(tmpCtx, longLivedRPCCtx)
+
+	// TODO: We should offer a version of StartTraceDataReceiveOp that starts the Span
+	// with as root, and links to the longLived context.
+	ctx := trace.NewContext(longLivedRPCCtx, trace.FromContext(tmpCtx))
```

Comment:
Thanks, I wasn't aware that this is the correct way to propagate the tags from parent while still having a trace for the individual operation. I will work on getting the TODO done.